### PR TITLE
Oops: Forgot matplotlib

### PR DESCRIPTION
**Pull request by zfrenchee**
@bgruening

### CODE REVIEW

**Comment by bgruening** [comment]
You need to bump the build number.

**Comment by zfrenchee** [comment]
@bgruening done. Unfortunately, the pypi package number is still 0.1.2... that might cause some confusion...

File: recipes/manormfast/meta.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 package:
   name: manormfast
-  version: "0.1.2"
+  version: "0.1.3"
```
Comment (bgruening) [inline recipes/manormfast/meta.yaml]: Thats not the build number ;)

Comment (zfrenchee) [inline recipes/manormfast/meta.yaml]: ?!?! Which one is that?

Comment (zfrenchee) [inline recipes/manormfast/meta.yaml]: Note: I didn't forget matplotlib in the pypi package, just in this file. I don't know what other number there is to change.

**Pull request merged**